static/styles/subscriptions.css

```diff
@@ -1021,6 +1021,10 @@ div.settings-radio-input-parent {
             }
         }
     }
+
+    #push-notifications-enabled {
+        margin: 25px auto;
```

Comment:
I think the margin of 25px is too much. Maybe we should use 20px which we used for `input-group` divs.